### PR TITLE
Cameramen live

### PR DESCRIPTION
**Pull request by TimYuenior**
UI for the live production for the cameramen.

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/gui/CameramenUIController.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
+package main.java.nl.tudelft.contextproject.gui;
```
Comment (Ettatabe) [inline src/main/java/nl/tudelft/contextproject/gui/CameramenUIController.java]: Shouldn't the package name be changed??

**Comment by bvobart** [comment]
The back button in the Cameraman UI does not work. Also, there is a problem with the initial values of Camera and Preset in the information about the live camera section.. They're initialized to -1.

**Comment by TimYuenior** [comment]
Back button is fixed, it was on the background.
Initial numbers are also fixed.

**Comment by bvobart** [comment]
After merging the dev branch into your own, the merge conflicts have been resolved and you now have the right package structure. When the Travis build passes, then this is ready for merging.

**Comment by Ettatabe** [comment]
This looks pretty good. This is ready for merging.

**Pull request merged**